UnityProject/Assets/Scripts/UI/Systems/AdminTools/PlayerManagePage.cs

```diff
@@ -64,6 +65,22 @@ public void OnDeputiseBtn()
 
 		public void OnRespawnButton()
 		{
+			if (quickRespawnToggle.isOn)
+			{
+				Occupation spawnOcc = new Occupation();
+				foreach (var connectedPlayer in PlayerList.Instance.AllPlayers)
+				{
+					if(connectedPlayer.Username != PlayerEntry.PlayerData.accountName) continue;
```

Comment:
as discussed on discord, find another thing to identify this account because account names aren't unique

Reply:
quick changes